.travis.yml

```diff
@@ -67,9 +66,7 @@ matrix:
       - tools
       - build-tools-26.0.2
 before_install:
-- rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm
-  && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm
-  install $TRAVIS_NODE_VERSION
+- if [[ "$PLATFORM" =~ android ]]; then nvm install 6; fi
```

Comment:
`nvm` is installed by default?
Why not via a ENV var any more? 
Why only for Android?

Reply:
Yes, `nvm` is installed per default in Trusty: https://docs.travis-ci.com/user/reference/trusty/#javascript-and-nodejs-images 
It's only needed for language "android" (or "$PLATFORM" called as the env variable) - because with `language: node_js` you can switch the version with the `node_js` YAML keyword (see https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) - android sadly does not understand this and needs the manual nvm installation.

I can restore the ENV variable, but it was used only once - that's why I removed it and used the `6` directly in the `before_install`

Reply:
Ok, I think we def need a comment explaining the android thing. Shame that this is so complicated.
Also I would prefer to go back to the ENV var, as it is really easy to miss stuff down here in before_install.

Reply:
Will add a small comment and restore the ENV variable